<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditColorMap</class>
 <widget class="QDialog" name="EditColorMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Color Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0,0">
   <item>
    <widget class="TransferFunctionHistogram" name="widgetHistogram" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Palette</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxPalette">
        <item>
         <property name="text">
          <string>Rainbow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rainbow extended</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Black body radiator</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blue to red diverging</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blue to tan diverging</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Purple to orange diverging</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green to purple diverging</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green to red diverging</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Highlight one</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelHighlight1">
        <property name="text">
         <string>Highlight one</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Highlight two</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelHighlight2">
        <property name="text">
         <string>Highlight two</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Color Coded Histogram</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Right click on the histogram to set clamping points.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransferFunctionHistogram</class>
   <extends>QWidget</extends>
   <header>TransferFunctionHistogram.h</header>
   <container>1</container>
   <slots>
    <signal>ClampHigh(double)</signal>
    <signal>ClampLow(double)</signal>
    <signal>ClampClear()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxPalette</sender>
   <signal>highlighted(int)</signal>
   <receiver>EditColorMap</receiver>
   <slot>HighlightedPalette(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditColorMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditColorMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetHistogram</sender>
   <signal>ClampHigh(double)</signal>
   <receiver>EditColorMap</receiver>
   <slot>ClampHigh(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetHistogram</sender>
   <signal>ClampLow(double)</signal>
   <receiver>EditColorMap</receiver>
   <slot>ClampLow(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetHistogram</sender>
   <signal>ClampClear()</signal>
   <receiver>EditColorMap</receiver>
   <slot>ClampClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>EditColorMap</receiver>
   <slot>ToggledColorCodedHistogram(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>533</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>HighlightedPalette(int)</slot>
  <slot>ClampHigh(double)</slot>
  <slot>ClampLow(double)</slot>
  <slot>ClampClear()</slot>
  <slot>ToggledColorCodedHistogram(bool)</slot>
 </slots>
</ui>
